bullet-journal/src/Components/Journal/VerticalLayout/Bullet.js

```diff
@@ -1,14 +1,93 @@
-import React, {useState, Fragment} from 'react'
+import React, {useState, useContext} from 'react'
+import {DayBulletsContext} from './DayView'
+import {WeekBulletsContext} from './WeekView'
 
 function Bullet(props) {
 
-    const [text, setText] = useState(props.bullet.text);
+    const [options, setOptions] = useState(false);
```

Comment:
you can use object destructuring here if you wanted instead:
`const { id, date, category, text, completed } = props.bullet;`